### PR TITLE
Adds test workflows to github in markdown

### LINKED ISSUE #202: Bug in test script: test 35
Test 35 involves disconnecting 1 of 2 guests, and expects the disband to be removed from the network screen.  This is incorrect, as there will still be one guest present.

This may be moot, as we've changed a host to always have a disband button..but the script still needs to be updated.

### LINKED ISSUE #246: Update test script to reflect app
When clearing the last song from the playlist, the app displays a call to action to add songs to the playlist.  This is not mentioned in the test scripts.

This applies to tests 13a, 13b, 19b, 19c, 19d, 20a, 20b, 20c, 20d.

Additionally, the "Playlist is empty" message only shows up on the phone that initiated the action called for in the test script.  This should be noted in the script

### LINKED ISSUE #247: Add reconnect to test 24
As a verification step, the guest should reconnect to the host in test 24.

### LINKED ISSUE #251: Put test script into git
Required:
Test script be in git.
Test script be viewable on github

Nice to have.
Test script have linkable tests.

### PR DESCRIPTION
**Pull request by taywrobel**
Adds a markdown version of the updated test
script and workflows to github, so that it
can be maintained by the entire team and reviewd
by beta participants.

Adds script to git - fixes #251
Removes incorrect disband check in test 35 - fixes #202
Includes check for toast when removing last song - fixes #246
Adds reconnect to test 24 - fixes #247

### CODE REVIEW

File: docs/testWorkflows.md [hunk 1]
```diff
@@ -0,0 +1,686 @@
...
+
+- *Complete Test 9*
+- Press the �Soundstream� icon on phone A
+- Press �Music Library� on phone A
+- Press �Add To Playlist� icon for one song from phone B�s music library on phone A
+- Press �Network� on phone B
+- Press the �Disconnect� button on phone B
+- Press �Create� on phone B
+- Press the �Play� icon in the playbar on phone A
+- Reconnect phone B to phone A's Network
```
Comment (theJenix) [inline docs/testWorkflows.md]: This should reference a test procedure (e.g. Complete Test 9) or be broken out into steps.

File: docs/testWorkflows.md [hunk 2]
```diff
@@ -0,0 +1,686 @@
...
+- Press the �Soundstream� icon on phone A
+- Press �Music Library� on phone A
+- Press �Add To Playlist� icon for one song on phone A
+- Press the �Soundstream� icon in the top corner on phone A
+- Press �Playlist� on phone A
+- Swipe the song off the playlist on phone A
+- Press the �Play� icon in the playbar on phone A
+
+### Expected Results: ###
+ Upon removal of last song, a toast appears instructing the user to add songs to the playlist. Upon pressing play, there is no song in the playlist and a toast appears saying �Playlist is empty�
```
Comment (theJenix) [inline docs/testWorkflows.md]: This needs to identify which phone the "Playlist is empty" toast appears on.

File: docs/testWorkflows.md [hunk 3]
```diff
@@ -0,0 +1,686 @@
...
+- *Complete Test 9*
+- Press �Music Library� on phone B
+- Press �Add To Playlist� icon for one song from phone A�s music library using phone B
+- Press the �Soundstream� icon in the top corner on phone B
+- Press �Playlist� on phone B
+- Swipe the song off the playlist on phone B
+- Press the �Play� icon in the playbar on phone B
+
+### Expected Results: ###
+ Upon removal of last song, a toast appears instructing the user to add songs to the playlist. Upon pressing play, there is no song in the playlist and a toast appears saying �Playlist is empty�
```
Comment (theJenix) [inline docs/testWorkflows.md]: This needs to identify which phone the "Playlist is empty" toast appears on.

**Comment by theJenix** [comment]
The tests that result in an empty playlist must indicate which phone (guest or host) will display the "Playlist is empty" message.

**Comment by reidbaker** [comment]
I would love to get rid of all the question marks. That should not hold off getting this pulled in but if it doesnt get fixed please make a ticket.

**Comment by dgreenhalgh** [comment]
@twrobel3,could we get the changes made and have this pulled in by this Sunday's meeting? I'd love to start looking forward at what's next.

**Comment by taywrobel** [comment]
Yeah, it's been a crazy week, but I can take care of it tomorrow.

On Fri, Aug 9, 2013 at 8:45 AM, David Greenhalgh
notifications@github.comwrote:

> @twrobel3 https://github.com/twrobel3,could we get the changes made and
> have this pulled in by this Sunday's meeting? I'd love to start looking
> forward at what's next.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/TheLastCrusade/SoundStream/pull/259#issuecomment-22403100
> .

## 

Taylor Wrobel
- www.linkedin.com/in/taylorwrobel/

**Comment by dgreenhalgh** [comment]
No worries, thanks man

**Comment by taywrobel** [comment]
@theJenix could you take a look and make sure that all the changes were addressed?  If so this should be good to push in.

**Comment by theJenix** [comment]
:+1: Looks good to me. :pushpin:

**Pull request merged**